### DISCUSSION TITLE
Resuming the workflow graphically or with code?

### DISCUSSION THREAD

**Discussion by PeterHuang-ST** [elsa-workflows/elsa-core#3209] [General]
This question is tie to question originally asked on https://github.com/elsa-workflows/elsa-core/discussions/3202

What would be the ways to performing resuming of the workflow? Would it be performed on a workflow instance or workflow registry display. When we write our code using c#, we would be able to see this in workflow Registry. For V2, would we be able to observe the operation and resume the workflow if the code is hand written in C#?

For V2, if we design a graphical interface, is there a way of embedded C# code into a activities? If so what would be the way to do it?

For V2, We would prefer to create complex activities and the workflow graphically and embed C# code for complex activities if possible. I do see an Inline activity under miscellaneous. Can we use this block to run C# code. But if that is not possible, we can write the whole workflow in C#. However, its more important if we can observe the workflow graphically if its written in C#. And be able to resume a work flow on error graphically if possible. If that is not possible graphically, please advise on how to perform resuming on a activity? Thanks.

**Comment by sfmskywalker** [comment]
Hi @PeterHuang-ST , please see [my comment](https://github.com/elsa-workflows/elsa-core/discussions/3202#discussioncomment-3164947) related to this question.
